awx/main/utils/analytics_proxy.py

```diff
@@ -102,8 +102,8 @@ def __init__(
         self,
         client_id: str,
         client_secret: str,
-        token_url: str,
-        scopes: list[str],
+        token_url: str = DEFAULT_OIDC_TOKEN_ENDPOINT,
+        scopes: list[str] = ['api.console'],
```

Comment:
I agree with SonarCloud - it's bad juju to use a `list` as a default, because the default itself is mutable. You should change this.

Reply:
fixed